src/model/config.go

```diff
@@ -3,14 +3,32 @@ package model
 import (
 	"fmt"
 	"os"
+	"syscall"
 )
 
+var expectedPermission os.FileMode = 0o644
+
+var IsOwnedByRoot = func(fi os.FileInfo) bool {
+	uid := fi.Sys().(*syscall.Stat_t).Uid
+	return uid == 0 // Check if the file is owned by root
+}
```

Comment:
It seems overkill to export and override this to make every other test pass.

Try using https://pkg.go.dev/testing#Testing

```suggestion
var isOwnedByRoot = func(fi os.FileInfo) bool {
	if testing.Testing() {
		return true // Always return true in test mode
	}
	uid := fi.Sys().(*syscall.Stat_t).Uid
	return uid == 0 // Check if the file is owned by root
}
```

The when testing isOwnedByRoot in unit tests of that package, override it to not always return true.

Reply:
Added here: https://github.com/canonical/rt-conf/pull/55/commits/4abf9abf62bb8a6096d4609e828942fcbd343892